sycl/source/detail/kernel_compiler/kernel_compiler_sycl.cpp

```diff
@@ -9,76 +9,18 @@
 #include "kernel_compiler_sycl.hpp"
 #include <sycl/exception.hpp> // make_error_code
 
-#include <sycl/detail/os_util.hpp>
-
-#include <ctime>
-#include <fstream>
-#include <random>
-#include <regex>
-#include <sstream>
-#include <stdio.h>
+#include <atomic>
+#include <numeric> // std::accumulate
 
-// For GCC versions less than 8, use experimental/filesystem.
-#if defined(__has_include) && __has_include(<filesystem>)
-#include <filesystem>
-namespace fs = std::filesystem;
-#elif defined(__has_include) && __has_include(<experimental/filesystem>)
-#include <experimental/filesystem>
-namespace fs = std::experimental::filesystem;
-#else
-#error "kernel_compiler sycl requires C++ filesystem support"
+#if SYCL_EXT_JIT_ENABLE
+#include "../jit_compiler.hpp"
```

Comment:
Nit: Can we use a non-relative path here?

Reply:
Yes!